content/articles/okta-identity-provider.markdown

```diff
@@ -0,0 +1,79 @@
+---
+title: Okta as an Identity Provider
+excerpt: How to use Okta to create and log in to accounts.
+categories:
+- Account
+---
+
+# Okta as an Identity Provider
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Using Okta as an identity provider streamlines the login experience for you and your team.
+
+## Prequisites
+
+In order to proceed with configuring login with SSO through Okta, you must:
+
+- Have administrator access to an Okta organization
+- Have administrator access to a DNSimple account
+- Have the Okta integration feature enabled on your DNSimple account
+
+## Supported Features
+
+- **SP-initiated Single Sign-On (SSO)**: This authentication flow occurs when the user attempts to log in to DNSimple from Okta.
```

Comment:
Do we support `JIT (Just In Time) Provisioning`?

Reply:
I would think we don't currently? As the provisioning of a DNSimple account is done during app assignment in Okta